policies/committer-policy.md

```diff
@@ -0,0 +1,96 @@
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by the [OMC] typically on the recommendation
+of the [OTC] (see the OpenSSL Bylaws).
+
+We welcome contributors who become domain experts in some part of
+the library (for example, low-level crypto) as well as generalists
+who contribute to all areas of the codebase.  All committers share
+the responsibility for the overall health of the project: aside from
+contributing quality features, committers are team players who fix bugs,
+address open issues, review community contributions, and improve tests
+and documentation.  Committers are also shepherds of the OpenSSL community
+and its code of conduct.
+
+To become a committer, start by contributing code.  Read our coding style,
+and get to know our build and test system.  Then, use the Github issue
+tracker, and our mailing lists find impactful ideas to work on.
+
+## How to maintain committer status?
+
+To maintain committer status, you must stay active in the project.  Refer
+to the OpenSSL Bylaws for details.
+
+In the unlikely and unfortunate event that your actions conflict with
+the project objectives or are otherwise disruptive, committer status
+may also be revoked by the OMC.
+
+## Approvals and code reviews
+
+All submissions must be reviewed and approved by at least two committers,
+one of whom must also be an OTC member.  If the author is also a committer
+then that counts as one of the reviews.  In other words:
+
+* OTC members need one approval from any committer
+* Committers need one approval from an OTC member
+* Contributors without commit rights need two approvals, including
+  one from an OTC member.
+
+An OMC member may apply an OMC-hold to a submission.  An OTC member may
+apply an OTC-hold to a submission.  An OMC-hold may be cleared by being
+removed by the member that put in place the hold or by a vote of the
+OMC.  An OTC-hold may be cleared by being removed by the member that put
+in place the hold or by a vote of the OTC.
+
+Approved submissions (outside of the automated release process and [NEWS]
+and [CHANGES] file updates) shall only be applied after a 24-hour delay from
+the approval (except for minor build and test breakage fix approvals).
+
+Contributors without commit rights cannot formally approve patches but
+are nevertheless welcome to comment on submissions and do technical
+reviews.  We always value another pair of eyes, and volunteering for
+reviews counts favourably towards becoming a committer.  As an author,
+we ask that you address all comments, even if you already have the
+necessary approvals.
+
+If you have trouble finding consensus on a difficult review, reach out to
+the OTC at `otc@openssl.org` (private, moderated) or the project
+at `openssl-project@openssl.org` (public, moderated).  On GitHub, you can
+target the OMC members with `@openssl/omc`, OTC members with `@openssl/otc`,
```

Comment:
"target" is a poor choice of word - "reach" would fit the context better.

Reply:
Agreed, target is the original terminology.  I've still to clean this up.